apps/server/src/modules/authentication/uc/dto/login.dto.ts

```diff
@@ -0,0 +1,7 @@
+export class LoginDto {
```

Comment:
Should we "publish" that type in an `src/modules/authentication/uc/dto/index.ts` file? So we can simplify imports.